rss.go

```diff
@@ -108,8 +108,8 @@ func newRssItem(i *Item) *RssItem {
 
 // create a new RssFeed with a generic Feed struct's data
 func (r *Rss) RssFeed() *RssFeed {
-	pub := anyTimeFormat(time.RFC822, r.Created, r.Updated)
-	build := anyTimeFormat(time.RFC822, r.Updated)
+	pub := anyTimeFormat(time.RFC1123Z, r.Created, r.Updated)
+	build := anyTimeFormat(time.RFC1123Z, r.Updated)
```

Comment:
The original issue/W3C validator requires an RFC822 time as per https://github.com/gorilla/feeds/issues/26

I'm confused as to why we're changing it to RFC1123Z?